ci-operator/step-registry/rosa/tests/rosa-tests-commands.sh

```diff
@@ -7,6 +7,7 @@ set -o pipefail
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 export REGION=${REGION:-}
 export TEST_PROFILE=${TEST_PROFILE:-}
+export COMPUTE_MACHINE_TYPE=${COMPUTE_MACHINE_TYPE:-"m5.xlarge"}
```

Comment:
Let's remove this line. It will cause all profiles compute machine type being override. Let's just define the COMPUTE_MACHINE_TYPE in yaml file. And also let's remove this settings from setup step.

Reply:
I remember it was the original design to expose COMPUTE_MACHINE_TYPE, so I do not think we should simply remove it. I advise checking whether the COMPUTE_MACHINE_TYPE is empty; if not, export it to override the default in profiles.